packages.json

```diff
@@ -23516,5 +23516,57 @@
     "description": "Wrappers over the Tkrzw Database Manager C++ library.",
     "license": "Apache-2.0",
     "web": "https://git.sr.ht/~ehmry/nim-tkrzw"
+  },
+  {
+    "name": "notcurses",
+    "url": "https://github.com/michaelsbradleyjr/nim-notcurses",
+    "method": "git",
+    "tags": [
+      "animate",
+      "animated",
+      "animates",
+      "animation",
+      "animations",
+      "character",
+      "characters",
+      "cli",
+      "clis",
+      "color",
+      "colors",
+      "command-line",
+      "console",
+      "consoles",
+      "curses",
+      "emoji",
+      "emojis",
+      "glyph",
+      "glyphs",
+      "grapheme",
+      "graphemes",
+      "graphic",
+      "graphical",
+      "graphics",
+      "image",
+      "images",
+      "interface",
+      "interfaces",
+      "multimedia",
+      "ncurses",
+      "notcurses",
+      "picture",
+      "pictures",
+      "terminal",
+      "terminals",
+      "tui",
+      "tuis",
+      "unicode",
+      "video",
+      "videos",
+      "widget",
+      "widgets"
```

Comment:
Okay, that's overdoing it... when you're adding plurals for each tag then you're going too far. You don't need this many, just:

* `tui`
* `library`
* `cli`

Is honestly enough.

Reply:
Alright, I'm happy to comply.

But, I wasn't on a wild tear; as I mentioned:

>  I've been working trough Notcurses' book/docs... there are terms (and variants of them) that come up frequently.

I tried out searches for various terms plural and singular, etc. on [nimble.directory](https://nimble.directory/) (not necessarily related to Notcurses) and found uneven results, so decided to expand on a base set of terms that I think are relevant to Notcurses, and so nim-notcurses:
```json
"tags": [
    "animate",
    "character",
    "cli",
    "color",
    "command-line",
    "console",
    "curses",
    "emoji",
    "glyph",
    "grapheme",
    "graphics",
    "image",
    "interface",
    "multimedia",
    "ncurses",
    "picture",
    "terminal",
    "tui",
    "unicode",
    "video",
    "widget"
 ]
```

If that's still too much, I'm happy to go with your suggestion of:
```json
"tags": [
    "cli",
    "library",
    "tui"
]
```

I was simply attempting to make nim-notcurses more find-able from [nimble.directory](https://nimble.directory/), given Notcurses touches on a lot of TUI capabilities; and because it's just package metadata, I didn't imagine it would be a problem.

I take no offense, and will modify my PR re: what you think is most suited re: nim-notcurses' `"tags": [...],"`.

Reply:
Personally I don't think we should be using these tags as a way to make things more searchable, the tags should be for making things more groupable. At the end of the day nimble.directory should maybe include keywords in package descriptions in its search, we shouldn't be adding many tens of tags to fix that for it.

Reply:
Fair enough, then I'll change it now, as you suggested, to:
```
"tags": [
    "cli",
    "library",
    "tui"
]
```

Reply:
Done.